tools/cell_census_builder/scripts/release/census_summary.py

```diff
@@ -2,10 +2,12 @@
 
 import cell_census
 import pandas as pd
-from cell_census_builder.globals import CENSUS_DATA_NAME
+
+from tools.cell_census_builder.globals import CENSUS_DATA_NAME, CENSUS_INFO_NAME
 
 if __name__ == "__main__":
```

Comment:
From a sustainability viewpoint, it would be much better if this code:
* had a real main() function
* returned a system result code (ie.,  `sys.exit(main())`)
* used argparse, so we could add additional features down the road

Reply:
For example, I think that it is _possible_ that future request might include a CLI param to specify what output format is desired:  text, markdown, html, ....

>  python census_summary_diff.py --output-format markdown new-tag old-tag

Reply:
It's certainly worth doing. I will work on it.

Reply:
I addd argparse and reformatted this into two separate subcommands: `summarize` and `diff`. LMK if it makes sense to you.